contrib/cirrus/integration_test.sh

```diff
@@ -24,6 +24,8 @@ case "${OS_RELEASE_ID}-${OS_RELEASE_VER}" in
     centos-7) ;&
     rhel-7)
         make install PREFIX=/usr ETCDIR=/etc
+        make podman-remote
```

Comment:
I don't think that line is necessary (looks like it would get it by `make` from ``unit_test.sh``  and `make install` (here).  But it's not a big deal / won't hurt anything.